packages/syntax-core/src/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,262 @@
+import { useState } from "react";
+import { action } from "@storybook/addon-actions";
```

Comment:
@somethiiing should we increase the zIndex in the examples to e.g. `1` so we don't see the show code buttons?
![image](https://github.com/Cambly/syntax/assets/127199/26e6b56f-9193-494b-ad9d-907c5d541221)

![image](https://github.com/Cambly/syntax/assets/127199/2e88ba69-919f-4aed-89de-343c3c73f684)